driver-sync/src/test/functional/com/mongodb/client/unified/ContextElement.java

```diff
@@ -42,28 +48,47 @@ static ContextElement ofError(final BsonDocument expectedError, final Exception
     }
 
     static ContextElement ofOutcome(final MongoNamespace namespace, final List<BsonDocument> expectedOutcome,
-                                           final List<BsonDocument> actualOutcome) {
+                                    final List<BsonDocument> actualOutcome) {
         return new OutcomeMatchingContextElement(namespace, expectedOutcome, actualOutcome);
     }
 
-    static ContextElement ofEvents(final String client) {
-        return new EventsMatchingContextElement(client);
+    static ContextElement ofEvents(final String client, BsonArray expectedEvents, List<CommandEvent> actualEvents) {
```

Comment:
Now the event context includes the expected and actual events, so that if there is a mismatch in the number of event, you can see what the actual ones were.  This helped debug one of the versioned API tests, where I thought it was an extra getMore but it was actually an extra killCursors